gtfs-validator/main/src/main/java/org/mobilitydata/gtfsvalidator/validator/StopsTooCloseValidator.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.validator;
+
+import static org.locationtech.spatial4j.context.SpatialContext.GEO;
+
+import java.lang.Double;
+import java.util.Collections;
+import java.util.List;
+import org.locationtech.spatial4j.distance.DistanceCalculator;
+import org.locationtech.spatial4j.distance.DistanceUtils;
+import org.locationtech.spatial4j.shape.Point;
+import org.locationtech.spatial4j.shape.ShapeFactory;
+import org.mobilitydata.gtfsvalidator.annotation.GtfsValidator;
+import org.mobilitydata.gtfsvalidator.annotation.Inject;
+import org.mobilitydata.gtfsvalidator.notice.NoticeContainer;
+import org.mobilitydata.gtfsvalidator.notice.StopsTooCloseNotice;
+import org.mobilitydata.gtfsvalidator.table.GtfsStop;
+import org.mobilitydata.gtfsvalidator.table.GtfsStopTableContainer;
+
+/**
+ * Validates that two stops are not too close to each other.
+ *
+ * <p>Generated notice: {@link StopsTooCloseNotice}.
+ */
+@GtfsValidator
+public class StopsTooCloseValidator extends FileValidator {
+  private static final int KILOMETER_TO_METER_CONVERSION_FACTOR = 1000;
+
+  // If stops are closer than 5m based on distance calculated, generate a notice
+  static final int DIST_BUFFER_METERS = 5;
+
+  /**
+   * Convert DIST_BUFFER_METERS to degrees of latitude. Converts a distance in the units of the
+   * radius to degrees (360 degrees are in a circle). Assumming spherical earth model
+   */
+  private static final double LAT_BUFFER = DistanceUtils.dist2Degrees(DIST_BUFFER_METERS,
+      DistanceUtils.EARTH_EQUATORIAL_RADIUS_KM* KILOMETER_TO_METER_CONVERSION_FACTOR);
```

Comment:
Nit:
DistanceUtils.EARTH_EQUATORIAL_RADIUS_KM * KILOMETER_TO_METER_CONVERSION_FACTOR)

Reply:
Fixed. Interestingly clang-format-path . changes it from ..KM * KILOMETER.. to ..KM* KILOMETER.. without the first space between the star